pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/db/dao/kit/BSPDummyKitDao.java

```diff
@@ -124,7 +124,7 @@ public Integer getRandomOncHistoryIdForStudy(String ddpInstanceName) {
                 stmt.setString(1, ddpInstanceName);
                 ResultSet rs = stmt.executeQuery();
                 if (rs.next()) {
-                    dbVals.resultValue = rs.getString(DBConstants.ONC_HISTORY_DETAIL_ID);
+                    dbVals.resultValue = rs.getInt(DBConstants.ONC_HISTORY_DETAIL_ID);
```

Comment:
The actual error comes in on L141 as a result of this being a String.  It should be an int.